go/viperutil/config.go

```diff
@@ -186,6 +187,14 @@ func LoadConfig() (context.CancelFunc, error) {
 	return registry.Dynamic.Watch(context.Background(), registry.Static, configPersistenceMinInterval.Get())
 }
 
+// isConfigFileNotFoundError checks if the error is caused because the file wasn't found.
+func isConfigFileNotFoundError(err error) bool {
+	if _, ok := err.(viper.ConfigFileNotFoundError); ok {
+		return true
```

Comment:
Should we use `errors.As` here to be more robust if there's future error chaining?

Reply:
@GuptaManan100 Yeah, I still think that's worth it then though?

Reply:
Okay